BUILD.boringssl.bazel

```diff
@@ -0,0 +1,29 @@
+# Builds and outputs crypto and ssl static libs (.lib or .a).
+# Doesn't output headers of those rules!!!
+filegroup(
+    name = "boringssl_built_libs",
+    srcs = [
+        "@boringssl//:crypto",
+        "@boringssl//:ssl",
+    ],
+)
```

Comment:
Have you ever tried just putting `@boringssl//:crypto` and `@boringssl//:ssl` in `srcs` of `boringssl_lib` below? Does `filegroup` somehow cause it to do something different? If you've never tried it I'd love to know what is different.

Reply:
It works!